frontend/src/components/assessment-creation/FilterMenu.tsx

```diff
@@ -1,12 +1,132 @@
-import React from "react";
-import { Button } from "@chakra-ui/react";
+import React, { SetStateAction } from "react";
+import {
+  Button,
+  Text,
+  Popover,
+  PopoverBody,
+  PopoverContent,
+  PopoverFooter,
+  PopoverTrigger,
+  Divider,
+  Flex,
+} from "@chakra-ui/react";
+import { Select } from "chakra-react-select";
 import { FunnelIcon } from "../../assets/icons";
 
-const FilterMenu = (): React.ReactElement => {
+export type Option = {
```

Comment:
can you move this into `src/types/SelectInputTypes.tsx`? and also refactor `SelectFormInputClassroom.tsx` to import that interface
```suggestion
export interface Option extends OptionBase {
```

Reply:
Done!